app/src/main/java/org/dasfoo/rover/android/client/grpc/task/GrpcConnection.java

```diff
@@ -88,7 +88,7 @@ public final boolean shutDownConnection() {
             return Boolean.TRUE;
         } catch (InterruptedException e) {
             if (BuildConfig.DEBUG) {
```

Comment:
Filtering by `BuildConfig.DEBUG` here seems to defeat the purpose of splitting `log_enable` into debug and release config.